pkg/collector/metric/utils.go

```diff
@@ -99,6 +99,9 @@ func isCounterStatEnabled(label string) bool {
 }
 
 func getNodeName() string {
+	if nodeName := os.Getenv("NODE_NAME"); nodeName != "" {
```

Comment:
? is it necessary in this PR?

Reply:
Yes, because we filter the events based on the node name due to performance reasons